### ISSUE TITLE
Add explicit FTP upload toggle

### ISSUE THREAD

**Issue by nhandler** [matfra/isitfoggy.today#8]
Right now, you need to unset `FTP_HOST` to disable the FTP logic in `timelapse`. It would be nice if there was an explicit `ENABLE_FTP` type boolean that could be set instead.

**Comment by matfra** [comment]
Yeah.also this option is very specific to my setup. I'm wondering if people would like to simply keep all the timelapses in the photos folder.also it could be neat to have a connector to Facebook, Instagram...

**Comment by matfra** [comment]
... to automatically upload the good videos

After reflection,it's probably better to do that  from the browser
